tests/parametric/test_telemetry.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+
+from utils import rfc, scenarios
+
+
+@scenarios.parametric
+@rfc("https://docs.google.com/document/d/1In4TfVBbKEztLzYg4g0si5H56uzAbYB3OfqzRGP2xhg/edit")
+class Test_Defaults:
+    """Clients should use and report the same default values for features."""
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                # Decrease the heartbeat/poll intervals to speed up the tests
+                "DD_TELEMETRY_HEARTBEAT_INTERVAL": "0.1",
+            }
+        ],
+    )
+    def test_library_settings(self, library_env, test_agent, test_library):
+        with test_library.start_span("test"):
+            pass
+        event = test_agent.wait_for_telemetry_event("app-started")
+        configuration = event["payload"]["configuration"]
+
+        configuration_by_name = {item["name"]: item for item in configuration}
+        for (apm_telemetry_name, value) in [
+            ("trace_sample_rate", "1.0"),
+            ("logs_injection_enabled", "false"),
```

Comment:
How can we exclude unsupported configs? For instance logs injection in the Go tracer.

Reply:
Great point! I wonder if we should add an "unsupported" option to the telemetry data. I will follow up with the telemetry team about this.

Reply:
@mehulsonowal do you think it makes sense to have an explicit "unsupported" entry in the data for settings?

Reply:
Hmm, so you mean have an additional field in the `client_library_configuration` table?

Reply:
If it's unsupported, the entry should probably be missing?